WasatchNET/EEPROM.cs

```diff
@@ -950,14 +981,15 @@ public bool write()
                 if (!ParseData.writeString(calibrationDate, pages[1], 48, 12)) return false;
                 if (!ParseData.writeString(calibrationBy, pages[1], 60, 3)) return false;
 
-                if (!ParseData.writeFloat(0.0f, pages[2], 21)) return false;
-
                 if (!ParseData.writeString(detectorName, pages[2], 0, 16)) return false;
                 if (!ParseData.writeUInt16(activePixelsHoriz, pages[2], 16)) return false;
                 // skip 18
                 if (!ParseData.writeUInt16(activePixelsVert, pages[2], 19)) return false;
                 if (!ParseData.writeUInt16((ushort)minIntegrationTimeMS, pages[2], 21)) return false; // for now
                 if (!ParseData.writeUInt16((ushort)maxIntegrationTimeMS, pages[2], 23)) return false; // for now
+
+                if (!ParseData.writeFloat(wavecalCoeffs[4], pages[2], 21)) return false;

```

Comment:
should you delete the two lines above this?

Reply:
Yes